utils/tf.py

```diff
@@ -8,9 +8,13 @@
 from datetime import datetime
 
 import tensorflow as tf
+from google.protobuf import text_format
+from tensorflow.python.framework import ops
+from tensorflow.core.framework import graph_pb2
 
 import utils.benchmark as bench_utils
 from utils.misc import advertise_aio
+tf.compat.v1.disable_eager_execution()
```

Comment:
I think this might be behind failing CI test for your PR (yolo model). You shouldn't disable eager exec for everything in AML just because your model requires that.

Reply:
Doesn't this change break functioning of some other models in zoo?